ert3/console/_console.py

```diff
@@ -353,6 +353,8 @@ def main() -> None:
     except ert.exceptions.ConfigValidationError as e:
         ert3.console.report_validation_errors(e)
         sys.exit(1)
+    except ert.exceptions.ExperimentError as e:
+        ert3.console.report_experiment_error(e)
```

Comment:
Will this still produce a non zero exit code?

Reply:
It does not, will fix.